.gitignore

```diff
@@ -1,2 +1,4 @@
+src/.env
 node_modules/
+webHookUrlForElon.txt
```

Comment:
add sample.env with only key's in it, so if anyone want to run in their local they can do it